ts/indicators/slow-stochastic.src.ts

```diff
@@ -0,0 +1,191 @@
+/* *
+ *
+ *  License: www.highcharts.com/license
+ *
+ *  !!!!!!! SOURCE GETS TRANSPILED BY TYPESCRIPT. EDIT TS FILE ONLY. !!!!!!!
+ *
+ * */
+
+'use strict';
+
+import H from '../parts/Globals.js';
+
+/**
+ * Internal types
+ * @private
+ */
+declare global {
+    namespace Highcharts {
+        class SlowStochasticIndicator extends StochasticIndicator {
+            public nameBase: string;
+            public init(): void;
```

Comment:
I assume that I should define here all methods that are overwritten, right? @bre1470

Reply:
Yes, except if you do a super call with all arguments (`apply(this, arguments)`).

Reply:
Ok, thanks! 👍